src/libsystemd/sd-journal/journal-file.c

```diff
@@ -25,6 +25,7 @@
 #include "journal-file.h"
 #include "lookup3.h"
 #include "memory-util.h"
+#include "missing_threads.h"
```

Comment:
Builds fail because of missing header. The `#include` can be just dropped, as the content of `missing_threads.h` is in `macro.h` in v252.